library/alloc/src/collections/btree/map/entry.rs

```diff
@@ -0,0 +1,475 @@
+use core::fmt::{self, Debug};
+use core::marker::PhantomData;
+use core::mem;
+
+use super::super::borrow::DormantMutRef;
+use super::super::node::{marker, Handle, InsertResult::*, NodeRef};
+use super::BTreeMap;
+
+use Entry::*;
```

Comment:
Should we make it more explicit and not use `use Entry::*`?

Reply:
I don't think so, in this file I think it's pretty clear that `Vacant` and `Occupied` are variants of `Entry`. I don't think adding `Entry::` in front of them is really worth it.

In `map.rs` it may make more sense to remove the glob import, given that there's only 2 uses each of the unqualified variant names